tools/install.sh

```diff
@@ -32,9 +32,9 @@ sed -i -e "/export PATH=/ c\\
 export PATH=\"$PATH\"
 " ~/.zshrc
 
-if [ "$SHELL" != "$(which zsh)" ]; then
+if [ "$SHELL" != "$(grep /zsh$ /etc/shells | tail -1)" ]; then
```

Comment:
I don't think `${PARAM:t}` works in basic `sh`, which `install.sh` runs under. How about `CURRENT_SHELL=$(expr "$SHELL" : '.*/\(.*\)'); if [ "$CURRENT_SHELL" != "zsh" ]; then ...`?